internal/handlers.go

```diff
@@ -31,3 +51,24 @@ func HeaderHandler(key, value string, next http.Handler) http.Handler {
 	}
 	return http.HandlerFunc(fn)
 }
+
+// InstrumentedHandler returns a handler that instruments http requests
+func InstrumentedHandler(totalRequests, responseStatus *prometheus.CounterVec, httpDuration *prometheus.HistogramVec, next http.Handler) http.Handler {
+	fn := func(w http.ResponseWriter, req *http.Request) {
+		path := req.URL.Path
+		timer := prometheus.NewTimer(prometheus.ObserverFunc(func(v float64) {
+			ms := v * millisecondsInSec // make microseconds
```

Comment:
v gets a float representing seconds, and we multiply by 1000, to turn it into milliseconds?

Reply:
Yep